bedrock/mozorg/templates/mozorg/contentservices/index.html

```diff
@@ -0,0 +1,70 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/base-resp.html" %}
+
+{% block page_title %}{{ _('Content Services') }}{% endblock %}
+{% block body_id %}contentservices{% endblock %}
+{% block body_class %}sand{% endblock %}
+
+{% block site_css %}{% endblock %}
+
+{% block extrahead %}
+  {% stylesheet 'contentservices' %}
```

Comment:
Is there a reason you're using the `extrahead` block here, but `page_css` in other files (like `start.html`)?